packages/venia-concept/src/AsyncWebpackPlugin.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * A Webpack plugin that takes an Async function to execute and
+ * halts webpacks execution till the Async function is not completed.
+ */
+class AsyncWebpackPlugin {
```

Comment:
This feels like it should go in `buildpack` instead of `venia-concept`.

Reply:
Makes sense. Will do that.